### PR TITLE
SQL Test Dialog

### PR DESCRIPTION
**Pull request by Kinplemelon**

### CODE REVIEW

File: src/components/Monaco.vue [hunk 1]
```diff
@@ -90,19 +89,67 @@ const initEditor = () => {
...
+            range: new monaco.Range(lineNumber, 1, lineNumber, endColumn),
+            options: {
+              after: {
+                content: createLineDecoration(lineContent),
+                inlineClassName: 'my-inline-decoration',
+              },
+            },
+          },
+        ])
+      }, 128)
```
Comment (ysfscream) [inline src/components/Monaco.vue:118]: Why 128ms?

Comment (Kinplemelon) [reply src/components/Monaco.vue:118]: just for fun 🤣

Comment (Kinplemelon) [reply src/components/Monaco.vue:118]: Should I change it?

Comment (ysfscream) [reply src/components/Monaco.vue:118]: No, it's cool 😂

**Review by ysfscream — APPROVED** [review]

**Pull request merged**